src/scss/base.scss

```diff
@@ -1,15 +1,16 @@
+
 @include oFontsInclude(BentonSans, normal);
 
-.o-ft-header {
+.o-header {
 	// 80 px for the primary container and 5 for an empty secondary or 25 for a secondary with content
 	min-height: 85px;
 	max-height: 125px;
 	font-family: oFontsGetFontFamilyWithFallbacks(BentonSans);
```

Comment:
Agreed. I'll show the design team what I built at the Guardian to abstract typography settings, maybe it can give them ideas around that matter from a design / ux perspective too.